Gems/EMotionFX/Code/EMotionFX/Source/AnimGraphNodeGroup.h

```diff
@@ -174,13 +174,26 @@ namespace EMotionFX
          */
         bool IsNameEditOngoing() const;
 
+        /**
+         * Returns the id of the parent AnimGraphNode to which this group belongs.
+         * If the group belongs to the root level, this function will return an invalid id.
+         * This function is used to make sure that groups can be filtered by level.
+         */
+        AnimGraphNodeId ParentNodeId() const;
+
+        /**
+         * Sets the id of the parent AnimGraphNode for this group.
+         */
+        void SetParentNodeId(AnimGraphNodeId nodeId);
```

Comment:
Since there is the setter `SetParentNodeId` please name the getter `GetParentNodeId` instead of `ParentNodeId`.

Reply:
Not relevant here, I mistakenly force pushed changes from a different PR here.